packages/inter-protocol/test/smartWallet/contexts.js

```diff
@@ -140,6 +140,19 @@ export const makeDefaultTestContext = async (t, makeSpace) => {
   };
 };
 
+/**
+ * @param {Awaited<ReturnType<typeof coalesceUpdates>>} state
+ * @param {Brand<'nat'>} brand
+ */
+export const purseBalance = (state, brand) => {
```

Comment:
this will only work for the invitation purse, right? call it `invitationPurseBalance`?

Reply:
Good idea. This was removed and reappeared from a faulty rebase so I'll just remove it.